src/pt/lermangaonline/src/eu/kanade/tachiyomi/extension/pt/lermangaonline/LerMangaOnlineUrlActivity.kt

```diff
@@ -0,0 +1,34 @@
+package eu.kanade.tachiyomi.extension.pt.lermangaonline
+
+import android.app.Activity
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import kotlin.system.exitProcess
+
+class LerMangaOnlineUrlActivity : Activity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val pathSegments = intent?.data?.pathSegments
+
+        if (pathSegments != null && pathSegments.size > 1) {
+            val intent = Intent().apply {
+                action = "eu.kanade.tachiyomi.SEARCH"
+                putExtra("filter", packageName)
+            }
```

Comment:
Your intent is missing a "query" extra, and your main source class is also missing an implementation for searching using a specific slug e.g. `slug:martial-peak`.

Reply:
Adding implementation for searching by a specific slug makes it possible to jump straight into the manga details view, since the app does so if there's only a single search result.

For implementations of searching by slug, you can search for `PREFIX_SLUG_SEARCH` or `PREFIX_ID_SLUG` in the codebase.

Reply:
fix: [9e837d2](https://github.com/keiyoushi/extensions-source/pull/657/commits/9e837d29f152af26a130cebfd19cb6e2811cc771)

Based: https://github.com/keiyoushi/extensions-source/blob/main/src/es/tumangaonline/src/eu/kanade/tachiyomi/extension/es/tumangaonline/TuMangaOnline.kt#L391